python-test.py

```diff
@@ -1,11 +1,11 @@
 print ("This is a tttsting code to check error catching")
 
-def simple_fn(range):
-    if range > 0:
+def simple_fn(range, seed):
+    if range >= 0 and seed == 1:
         for i in range(range):
```

Comment:
Consider renaming the 'range' parameter to avoid shadowing the built-in 'range' function.